packages/core/src/connectivity/scp-cf/proxy-util.spec.ts

```diff
@@ -232,6 +232,11 @@ describe('proxy-util', () => {
       );
       expect(parseProxyEnv('user:password@some.host')?.host).toBe('some.host');
 
+      expect(parseProxyEnv('some-host')?.host).toBe('some-host');
+      expect(parseProxyEnv('us-er:pass-word@some-host')?.host).toBe(
+        'some-host'
+      );
```

Comment:
Could you also make a test with `us-er:pass-word@some-host:1234` where also the port is correctly asserted.

Reply:
Done.